.gitignore

```diff
@@ -6,10 +6,12 @@ modules/livecd_extras
 modules/hybridcd_extras
 modules/3rdparty
 modules/[Aa][Hh][Kk]_[Tt]ests
+.vs
 .cache
 .cproject
 .DS_Store
 .project
 .settings
 .vscode
 sdk/tools/winesync/winesync.cfg
+out
```

Comment:
Please revert unrelated changes here.

Reply:
Is reverted.

Reply:
Why don't you add the .vs folder when working with Visual Studio?

Reply:
I asked to revert it because it's unrelated to the translation. But feel free to propose these changes in a separate PR.